packages/@sanity/components/src/buttons/ButtonsCollection.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable complexity */
+import PropTypes from 'prop-types'
+import React from 'react'
+import styles from 'part:@sanity/components/buttons/collection-style'
+
+export default class ButtonsCollection extends React.PureComponent {
```

Comment:
Idiomatically, should perhaps be named with singular `button` – as in `ButtonCollection`

Reply:
This belongs to #824